### PR TITLE
Issue #28 Fix, check for tika.jar

### PR DESCRIPTION
**Pull request by andymanning**

### CODE REVIEW

File: tika/server.go [hunk 1]
```diff
@@ -91,6 +90,13 @@ func NewServer(jar, port string) (*Server, error) {
 	if jar == "" {
 		return nil, fmt.Errorf("no jar file specified")
 	}
+
+	//Check if the jar file exists
+	_, err := os.Stat(jar)
+	if os.IsNotExist(err) {
+		return nil, fmt.Errorf("jar file [%s] does not exist", jar)
+	}
```
[the remark this answers is not in the document]
Comment (Nimesh199398) [reply tika/server.go:98]: Experia help u

**Review by Nimesh199398 — APPROVED** [review]

File: tika/server.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nimesh199398) [reply tika/server.go:98]: Networking

[the remark this answers is not in the document]
Comment (Nimesh199398) [reply tika/server.go:98]: Help time spend with you

[the remark this answers is not in the document]
Comment (Nimesh199398) [reply tika/server.go:98]: > ```suggestion
> 	// Check if the jar file exists.
> 	if _, err := os.Stat(jar); os.IsNotExist(err) {
> 		return nil, fmt.Errorf("jar file %q does not exist", jar)
> 	}
> ```

Networking events across your

**Review by tbpg — APPROVED** [review]

**Pull request merged**